### DISCUSSION TITLE
Stopping Concourse Pipeline scanning continuously

### DISCUSSION THREAD

**Discussion by bagyaraj-palanisamy** [concourse/concourse#8982] [Help & Support]
Hi Team,

We are using Concourse CI for build and deployment for my applications. My pipeline continuously scanning the repository because of that I am getting Too Many Requests error.

Please could you help me to stop continuous scanning configuration in the pipeline configuration. 

I have shared the error in below for your reference. 

**resource script '/opt/resource/check []' failed: exit status 1
stderr:
failed to fetch digest: 429 Too Many Requests**

Your support much appreciated, thank you in advance 

Regards,
Bagyaraj P

**Comment by marco-m-pix4d** [comment]
This is the check interval, which defaults to 1 minute. If you get 429, I assume that you have multiple pipelines referring to the same git repo.

You can change it in two places:

- globally, for ALL resources: https://concourse-ci.org/performance-tuning.html#concourse_resource_checking_interval
- per pipeline, per resource: https://concourse-ci.org/resources.html#schema.resource.check_every.

Example:

```
- name: foo
  type: git
  check_every: 10m
  source:
    uri: ...
```

But I suggest instead to use webhooks, see https://concourse-ci.org/resources.html#schema.resource.webhook_token
